tests/unittest/_torch/visual_gen/multi_gpu/test_visual_gen_mapping.py

```
@@ -289,11 +301,8 @@ def _logic_allreduce_over_tp_group(rank, world_size):
 def _logic_attn2d_mesh_rank_and_group(rank, world_size):
     """attn2d_mesh_rank aliases cp_rank and attn2d_mesh_group works for collectives (2x2 mesh).
 
-    Default order cfg-tp-cp-ulysses with attn2d 2x2 (cp_size=4), cfg=1, tp=1, ulysses=1:
-        Rank 0: cp=0  (attn2d_mesh_rank=0)
-        Rank 1: cp=1  (attn2d_mesh_rank=1)
-        Rank 2: cp=2  (attn2d_mesh_rank=2)
-        Rank 3: cp=3  (attn2d_mesh_rank=3)
+    Mesh ``cfg-tp-cp_row-cp_col-ulysses`` with attn2d 2×2, cfg=1, tp=1, ulysses=1:
```

Ruff (0.15.13)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)